container/kvm/sync.go

```diff
@@ -0,0 +1,256 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// +build linux
+// +build amd64 arm64 ppc64el
+
+package kvm
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/juju/errors"
+	"github.com/juju/utils/series"
+
+	"github.com/juju/juju/environs/imagedownloads"
+	"github.com/juju/juju/environs/simplestreams"
+	"github.com/juju/juju/juju/paths"
+)
+
+const (
```

Comment:
yes, I believe I was, just making a very unclear comment but I believe I meant: convert this to comment newline const declaration